src/objective/regression_objective.hpp

```diff
@@ -343,6 +360,25 @@ class RegressionPoissonLoss: public ObjectiveFunction {
 
   bool BoostFromAverage() const override { return true; }
 
+  bool GetCustomAverage(double *initscore) const override {
```

Comment:
I don't know what is this for.

Reply:
It makes the boost_from_average option behave so that the internal score is initialized to the value that corresponds to the average label on the output.

Reply:
I got it.